web/src/constants/demos/pokeshop.postman.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * Moved from JSON files to TS files, as JSONs are not working in composite projects.
+ * @see {@link https://github.com/TypeStrong/ts-loader/issues/905}
+ */
+export default {
+  info: {
+    _postman_id: '910a51e7-5e97-4e2e-ba77-5f6d8c7625dd',
+    name: 'Poke Micro API - DEV',
+    schema: 'https://schema.getpostman.com/json/collection/v2.1.0/collection.json',
+    _exporter_id: '326743',
+  },
+  item: [
+    {
+      name: 'List',
+      request: {
+        method: 'GET',
+        header: [],
+        url: {
+          raw: 'http://demo-pokemon-api.demo.svc.cluster.local/pokemon?take=20&skip=0',
```

Comment:
We can also update the urls to match the configuration instead of hardcoded values